integrated_channels/api/v1/blackboard/urls.py

```diff
@@ -0,0 +1,12 @@
+"""
+    url mappings for integrated_channels/v1/moodle/
```

Comment:
"general cleanup" probably includes this reference, but just in case!

Reply:
😡 I was so sure I got them all. Hahaha yes general cleanup did refer to things like this